cmplibrary/src/test/java/com/sourcepoint/cmplibrary/data/network/util/HttpUrlManagerTest.kt

```diff
@@ -420,16 +439,18 @@ class HttpUrlManagerTest {
             authId = null,
             accountId = 1212,
             propertyId = 12,
+            nonKeyedLocalState = fakeNonKeyedLocalState,
```

Comment:
just use something like this  JsonObject(mapOf("a" to "a")) and check the condition

Reply:
the same for localState